html/modules/custom/reliefweb_job_tagger/src/Form/RwJobTaggerCategory.php

```diff
@@ -517,4 +540,39 @@ public function getMostRelevantTermsFromEs(
     return $vocabularies;
   }
 
+  /**
+   * Get similar jobs.
+   */
+  protected function getSimilarJobs(NodeInterface $node) {
```

Comment:
This doesn't use the similarity scores from ES, is that on purpose?

Reply:
see https://github.com/UN-OCHA/ocha_ai/pull/47

Reply:
Switch to normalized score